### DISCUSSION TITLE
vtiname can be different between 2 conns, the first as "main" and the second as "also"?

### DISCUSSION THREAD

**Discussion by marcogaravello** [strongswan/strongswan#2162] [Q&A] [answered]
Hello there,

the question is as per subject, I'll try to clarify better pasting here the configuration I'm using in ipsec.conf file

```
conn FOOBAR-1
    keyexchange=ikev1
    leftupdown=/etc/strongswan.d/ipsec-vti-helper --mtu:1418 --vtiname:FOOBAR
    mark=1120
    leftsubnet=10.1.180.0/24
    right=<public_ip>
    rightid=<public_ip>
    rightsubnet=10.1.181.0/24
    dpddelay=15
    dpdtimeout=45
    dpdaction=restart
    auto=start

conn FOOBAR-2
    also=FOOBAR-1
    leftupdown=/etc/strongswan.d/ipsec-vti-helper --mtu:1418 --vtiname:FOOBAR
    mark=1120
    rightsubnet=10.1.188.0/24
    auto=start
```

As you can notice, as vtiname I left FOOBAR for both of them.
But could I use FOOBAR-1 for the first, and FOOBAR-2 for the second? Is it supported?

Thanks in advance

**Comment by tobiasbrunner** [comment, marked as answer]
> But could I use FOOBAR-1 for the first, and FOOBAR-2 for the second? Is it supported?

You have to ask whoever wrote `ipsec-vti-helper` (or look at the script yourself) as that's not from us. And you probably have to use different marks.

**Comment by marcogaravello** [comment]
Hello,

thanks for your answer.
I didn't realize script was a custom part, thanks for the hint.
Regarding marks: why do you expect 2 different?

**Reply by tobiasbrunner** [reply]
I'm not sure if that's the case. It might not be. However, you might actually not even be able to create multiple VTIs in the first place. Because according to [our own documentation](https://docs.strongswan.org/docs/5.9/features/routeBasedVpn.html), one limitation of VTIs is that their local and remote IP must be unique. You might want to use XFRM interfaces instead, which don't require IPs at all.
